membership-common/src/main/scala/com/gu/memsub/ProductFamily.scala

```diff
@@ -139,7 +138,6 @@ object Benefit {
       (id == Weekly.id).option(Weekly)
 
   sealed trait MemberTier extends Benefit
-  sealed trait FreeMemberTier extends MemberTier
```

Comment:
I removed a few of the `Free...` things, but I tried to avoid doing to much in this PR, the next PR will be imminent.